examples/interaction/tooltips/tooltip_description.py

```diff
@@ -1,9 +1,19 @@
+''' A visualization of Multichoice tooltips in bokeh.models.
+This example demonstrates user input text box where
+the user can input from al list of various options of elements to choose one or more of them,
+and then the clear the choices as well.
```

Comment:
Please change the line breaks to make the line lengths more consistent with each other (and remember not trailing spaces or the linter will complain)